### PR TITLE
Add praduns-Farmington file to Master

### LINKED ISSUE #14: Create a file for praduns Farmington, NY
- [ ] Create a branch
- [ ] Add commits
- [ ] Open a pull request
- [ ] Collaborate
- [ ] Receive Approval :+1: 
- [ ] Merge :shipit: 
- [ ] Eat Cookies

### PR DESCRIPTION
**Pull request by praduns**
closes #14

### CODE REVIEW

File: praduns-Farmington.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+### Farmington, NY
+
+#### Places to Go
+- Canandaigua Lake
+- Finger Lakes Casino
+
+#### Places to Eat
+- Mark's Pizzeria
+
```
Comment (thomaslinder) [inline praduns-Farmington.md:11]: Any other places to eat.  Pizza upsets my stomach :-1:

Comment (thomaslinder) [inline praduns-Farmington.md:11]: Any other places to eat.  Pizza upsets my stomach :-1:

File: praduns-Farmington.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+### Farmington, NY
+
+#### Places to Go
+- Canandaigua Lake
+- Finger Lakes Casino
+
+#### Places to Eat
+- Mark's Pizzeria
```
Comment (thomaslinder) [inline praduns-Farmington.md:9]: Are there any other places to eat?

Comment (thomaslinder) [inline praduns-Farmington.md:9]: Are there any other places to eat?

File: praduns-Farmington.md (hunk 1 above)
Comment (praduns) [reply praduns-Farmington.md:11]: Otto Tomatos (Italian)

Comment (thomaslinder) [inline praduns-Farmington.md:11]: Any other places to eat.  Pizza upsets my stomach :-1:

**Comment by praduns** [comment]
@cshing1 Does this request look good to :shipit: ?

**Comment by cshing1** [comment]
> @cshing1 Does this request look good to :shipit: ?

@praduns :+1:

**Pull request merged**